upup/pkg/fi/cloudup/openstack/image.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openstack
+
+import (
+	"fmt"
+
```

Comment:
remove additional new line

Reply:
Which line are you referring to?

Reply:
you have additional whitespace in your src code

Reply:
I have ran `go fmt` and `goimports` on the file both from an IDE and the formatting scripts they provide. There were no changes made to the file.

Reply:
@foamdino this whitespace is Go convention to have a separation between core library and third party dependencies